deepchem/data/data_loader.py

```diff
@@ -876,7 +876,16 @@ def _featurize_shard(self,
       Boolean values indicating successful featurization for corresponding
       sample in the source.
     """
-    features = [elt for elt in self.featurizer(shard[self.mol_field])]
+    pos_cols = ['pos_x', 'pos_y', 'pos_z']
```

Comment:
Are these standard names for the position columns or are they specific to the qm9 dataset?